esp/services/ws_dfu/ws_dfuService.cpp

```diff
@@ -2351,6 +2351,28 @@ void CWsDfuEx::doGetFileDetails(IEspContext &context, IUserDescriptor *udesc, co
         }
     }
 
+#if 1
+    unsigned maxSkew, minSkew, maxSkewPart, minSkewPart;
+    if (df->getSkewInfo(maxSkew, minSkew, maxSkewPart, minSkewPart, true))
+    {
+        IEspDFUFileStat& Stat = FileDetails.updateStat();
+        if (version >= 1.38)
+        {
+            Stat.setMinSkewInt64(minSkew);
+            Stat.setMaxSkewInt64(maxSkew);
+        }
+        if (version >= 1.52)
+        {
+            Stat.setMinSkewPart(minSkewPart);
+            Stat.setMaxSkewPart(maxSkewPart);
+        }
+
+        VStringBuffer minSkewString("-%.2f", ((double)minSkew)/100);
```

Comment:
Should the minus be added if minSkew = 0?

Reply:
@jakesmith did you miss this comment?

Reply:
I did miss it, thanks. I agree should be removed.
I'll change.

Reply:
@wangkx - have now amended to avoid adding minus if no skew.